ui/admin/tests/acceptance/workers/create-tags-test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { visit, currentURL, click, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+
+module('Acceptance | workers | worker | create-tags', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  const SAVE_BUTTON_SELECTOR = '[type="submit"]';
+  const CANCEL_BUTTON_SELECTOR = '.rose-form-actions [type="button"]';
+  const MANAGE_DROPDOWN_TOGGLE =
+    '[data-test-manage-worker-dropdown] div button';
+  const CREATE_TAGS_BUTTON_SELECTOR =
+    '[data-test-manage-worker-dropdown] div:first-child a';
+  const KEY_INPUT_SELECTOR = '[name="api_tags"] tr td:first-child input';
+  const VALUE_INPUT_SELECTOR = '[name="api_tags"] tr td:nth-child(2) input';
+  const ADD_INPUT_SELECTOR = '[name="api_tags"] tr td:last-child button';
+  const DISCARD_CHANGES_DIALOG = '.rose-dialog';
+  const DISCARD_CHANGES_DISCARD_BUTTON =
+    '.rose-dialog-footer .rose-button-primary';
+  const DISCARD_CHANGES_CANCEL_BUTTON =
+    '.rose-dialog-footer .rose-button-secondary';
+
+  const instances = {
+    scopes: {
+      global: null,
+    },
+    worker: null,
+  };
+
+  const urls = {
+    workers: null,
+    worker: null,
+    tags: null,
+    createTags: null,
+  };
+
+  hooks.beforeEach(function () {
+    instances.scopes.global = this.server.create('scope', { id: 'global' });
+    const scope = instances.scopes.global;
+    instances.worker = this.server.create('worker', {
+      scopeId: scope.id,
```

Comment:
This needs to be the entire scope model passed in, not just the ID

```suggestion
      scope,
```

Reply:
Thanks for spotting that. I updated it in a few other places as well.